Lib/test/_test_multiprocessing.py

```diff
@@ -6801,6 +6801,41 @@ def test_child_sys_path(self):
         self.assertEqual(child_sys_path[1:], sys.path[1:])
         self.assertIsNone(import_error, msg=f"child could not import {self._mod_name}")
 
+    def test_std_streams_flushed_after_preload(self):
+        # gh-135335: Check fork server flushes standard streams after
+        # preloading modules
+        if multiprocessing.get_start_method() != "forkserver":
+            self.skipTest("forkserver specific test")
+
+        # Create a test module in the temporary directory on the child's path
+        # TODO: This can all be simplified once gh-126631 is fixed and we can
+        #       use __main__ instead of a module.
+        dirname = os.path.join(self._temp_dir, 'preloaded_module')
+        init_name = os.path.join(dirname, '__init__.py')
+        os.mkdir(dirname)
+        with open(os.path.join(init_name), "w") as f:
+            cmd = '''if 1:
+                import sys
+                print('stderr', file=sys.stderr)
+                print('stdout', file=sys.stdout)
```

Comment:
By default, print() adds a newline and the io module flushs the stream in this case. To test your fix, you should skip the newline:

```suggestion
                print('stderr', end='', file=sys.stderr)
                print('stdout', end='', file=sys.stdout)
```

Reply:
Hmm. Good point. However, the test does fail without the fix. I just rechecked and confirmed. Odd. I'll take the suggestion regardless, thanks.

Reply:
Maybe you've missed this? I can't see this change

Reply:
Argh, I did indeed, thanks very much for catching that!